roles/mirror_ocp_release/tasks/registry.yml

```diff
@@ -27,12 +27,15 @@
 
   rescue:
     - name: "Mirror release images to local registry"
-      ansible.builtin.command: >
+      ansible.builtin.command: >-
         {{ mor_cache_dir }}/{{ mor_version }}/oc adm release mirror
         --registry-config={{ mor_auths_file }}
         --from={{ ocp_release_data['container_image'] | quote }}
         --to-release-image={{ mor_registry_url }}/{{ mor_registry_path }}:{{ mor_version }}
         --to={{ mor_registry_url }}/{{ mor_registry_path }}
+        {% if mor_version is version("4.14", ">=") %}
```

Comment:
Thanks Tony, from our conversation about this yesterday, AI still uses `ImageContentSourcePolicy` in 4.14, so I wonder if this option would print both outputs of `ImageContentSourcePolicy` and `ImageDigestMirrorSet` so we can decide which one to use at the end?

Reply:
Great observation, that would be a problem :crying_cat_face:  This change generates only the IDMS. A better approach is to make this role to have the option of using one or the other and only allowing IDMS for 4.14 and above. Let me change this.

Reply:
This change should address it, that is IPI (4.14+) Will request IDMS (https://softwarefactory-project.io/r/c/dci-openshift-agent/+/31736), while ABI will not